kolibri/core/assets/src/views/StorageNotification.vue

```diff
@@ -114,17 +125,21 @@
         return message;
       },
       closeBanner() {
-        // TODO: Store preference after closure
+        localStorage.setItem('download_removed', this.lastDownloadRemoved);
+        localStorage.setItem('last_synced', this.lastSynced);
         this.bannerOpened = false;
+
         if (this.previouslyFocusedElement) {
           this.previouslyFocusedElement.focus();
         }
       },
       manageChannel() {
         this.$router.push('/');
       },
-      toggleBanner() {
+      validateDeviceStatus() {
```

Comment:
Not sure if this is an appropriate name to to use for this method; it seems a little misleading based on its logic. Maybe maintaining `toggleBanner` would suffice?

Reply:
I also think the logic therein is still be flawed. If `showBanner` is true [which I presume is meant to display the banner], setting `bannerOpened` to `false` would dismiss it instead. I think it should be the other way round. 
if my assumption is correct, a simple `this.bannerOpened = this.showBanner` would be enough to display the banner when the condition to show a banner is met.

Reply:
Part of the reason I mentioned the Vue-use composable for local storage was that it should also be reactive to changes. So if the component was using it, the computed prop `showBanner` would change to false, when setting `download_removed` and `last_synced` in local storage. That should make managing `bannerOpened` no longer needed